docs/collector.service.md

```diff
@@ -18,6 +18,10 @@ Example: `--collector.service.services-where="Name='windows_exporter'"`
 
 Example config win_exporter.yml for multiple services: `services-where: Name='SQLServer' OR Name='Couchbase' OR Name='Spooler' OR Name='ActiveMQ'`
 
+### `--collector.service.useAPI`
```

Comment:
The flag should not have capital letters, in Go the variable should be useAPI because of good naming practices, but the flag should be something like --collector.service.use-api

Reply:
I guess it should be also changed in the code